### PR TITLE
ci(build): add gh action pipelines

### PR DESCRIPTION
**Pull request by mmpetarpeshev**

### CODE REVIEW

File: .github/workflows/prod-docker-mainnet.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/mdw-frontend:mainnet # has to be changed
+          tags: |
+            type=raw,value=latest,enable=${{ endsWith(GitHub.ref, 'main') }}
+            type=raw,value=${{ steps.release.outputs.tag_name }},enable=${{ endsWith(GitHub.ref, 'main') }}
+            type=ref,event=tag
+            type=ref,event=pr
+
+      - name: Build and push docker image
```
Comment (venimus) [inline .github/workflows/prod-docker-mainnet.yml:69]: make the same for testnet

File: .github/workflows/prod-docker-mainnet.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: nginx.Dockerfile
+          build-args: |
+            NUXT_APP_NODE_URL: https://mainnet.aeternity.io/v3
+            NUXT_APP_NODE_WS: wss://mainnet.aeternity.io/mdw/websocket
+            NUXT_APP_MDW_URL: https://mainnet.aeternity.io/mdw
+            NUXT_APP_OTHER_DEPLOYMENTS: TESTNET@https://explorer.testnet.aeternity.io
+            NUXT_APP_NETWORK_NAME: MAIN NET
```
Comment (venimus) [inline .github/workflows/prod-docker-mainnet.yml:82]: MAINNET

File: .github/workflows/prod-docker-mainnet.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            NUXT_APP_ENABLE_FAUCET: false
+            NUXT_APP_FAUCET_API: https://testnet.faucet.aepps.com/account
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
```
Comment (venimus) [inline .github/workflows/prod-docker-mainnet.yml:86]: tag should be `master-mainnet` or `<sha>-mainnet`

File: .github/workflows/prod-docker-mainnet.yml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      - name: Extract metadata for docker
+        if: |
+          github.event_name == 'push' ||
+          startsWith(github.ref, 'refs/tags/v')
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/mdw-frontend:mainnet # has to be changed
+          tags: |
+            type=raw,value=latest,enable=${{ endsWith(GitHub.ref, 'main') }}
```
Comment (venimus) [inline .github/workflows/prod-docker-mainnet.yml:64]: the main branch is `master`

File: .github/workflows/staging-docker-testnet.yml [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+
+      - name: Extract metadata for docker
+        if: github.event_name == 'pull_request' && github.event.action == 'opened' || github.event.action == 'synchronize'
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/mdw-frontend-testnet
```
Comment (venimus) [inline .github/workflows/staging-docker-testnet.yml:51]: i don't think we have such thing

File: .github/workflows/staging-docker-mainnet.yml [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+
+      - name: Extract metadata for docker
+        if: github.event_name == 'pull_request' && github.event.action == 'opened' || github.event.action == 'synchronize'
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/mdw-frontend-mainnet
```
Comment (venimus) [inline .github/workflows/staging-docker-mainnet.yml:51]: i don't think we have that

File: .github/workflows/staging-docker-testnet.yml [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+          echo "::set-output name=git-sha::$(git rev-parse --short HEAD)"
+      
+      - name: Build and push docker image
+        if: github.event_name == 'pull_request' && github.event.action == 'opened' || github.event.action == 'synchronize'
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: nginx.Dockerfile
+          build-args: |
+            NUXT_APP_NETWORK_NAME: "TESTNET v0.7.1"
```
Comment (venimus) [inline .github/workflows/staging-docker-testnet.yml:71]: version should not be included if it won't match

File: .github/workflows/prod-docker-mainnet.yml (hunk 2 above)
Comment (mmpetarpeshev) [reply .github/workflows/prod-docker-mainnet.yml:82]: I just get the one that you send me in rocket chat. I thought the same, jut wanted to verify with review.

File: .github/workflows/prod-docker-mainnet.yml (hunk 3 above)
Comment (mmpetarpeshev) [reply .github/workflows/prod-docker-mainnet.yml:86]: ye , something for clarifying mainnet on every release?

File: .github/workflows/prod-docker-mainnet.yml (hunk 4 above)
Comment (mmpetarpeshev) [reply .github/workflows/prod-docker-mainnet.yml:64]: fixed.

File: .github/workflows/staging-docker-testnet.yml (hunk 5 above)
Comment (mmpetarpeshev) [reply .github/workflows/staging-docker-testnet.yml:51]: you mean the image, if we dont have, will create one.

File: .github/workflows/staging-docker-mainnet.yml (hunk 6 above)
Comment (mmpetarpeshev) [reply .github/workflows/staging-docker-mainnet.yml:51]: ye , wrong replace , will fix it

**Pull request closed without merging**